### PR TITLE
[Consignments] Adds the final submission screen

### PR DESCRIPTION
**Pull request by orta**
This adds the a screen that is the final step in the submission process. Handles all the UI work, but none of the logic towards shipping the final build.

Made a nice GIF of my SVGs for testing too:

### CODE REVIEW

File: src/lib/Components/Consignments/Screens/FinalSubmissionQuestions.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    // no arguments.
+    const animate = LayoutAnimation.easeInEaseOut as any
+    animate()
+
+    this.setState({
+      editionInfo: this.state.editionInfo ? null : {},
+    })
+  }
+
+  updateSigned = () => this.setState({ signed: this.state.signed ? false : true })
```
Comment (sweir27) [inline src/lib/Components/Consignments/Screens/FinalSubmissionQuestions.tsx]: Could you do `{ signed: !Boolean(this.state.signed) }`? (or just `!this.state.signed` if it's coming back as truthy already)

File: src/lib/Components/Consignments/Screens/FinalSubmissionQuestions.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+          </View>
+          <View style={{ padding: 10 }}>
+            <View style={{ flexDirection: "row", paddingVertical: 6, alignItems: "center" }}>
+              <BodyText style={{ paddingLeft: 10, flex: 1, textAlign: "left" }}>Is this an edition?</BodyText>
+              <Toggle selected={!!this.state.editionInfo} left="NO" right="YES" onPress={this.updateEdition} />
+            </View>
+
+            {this.state.editionInfo
+              ? <View style={{ flexDirection: "row", paddingVertical: 6 }}>
+                  <Text text={{ placeholder: "Edition Size", keyboardType: "phone-pad" }} style={{ margin: 10 }} />
```
Comment (sweir27) [inline src/lib/Components/Consignments/Screens/FinalSubmissionQuestions.tsx]: Can we do some client-side validation here? We only accept edition sizes that are numbers in convection (`edition_number` is a string to accommodate values like `12a/100`).

File: src/lib/Components/Consignments/Components/__tests__/TextInput-tests.tsx [hunk 3]
```diff
@@ -9,6 +9,6 @@ it("shows an activity indicator when searching ", () => {
 })
 
 it("does not have an activity when searching ", () => {
-  const component = renderer.create(<Text text={{ value: "My mocked" }} searching={true} />).toJSON()
+  const component = renderer.create(<Text text={{ value: "My mocked" }} searching={false} />).toJSON()
```
Comment (orta) [inline src/lib/Components/Consignments/Components/__tests__/TextInput-tests.tsx]: oh yeah, this shouldn't have been commited 👍

File: src/lib/Components/Consignments/Screens/FinalSubmissionQuestions.tsx (hunk 1 above)
Comment (orta) [inline src/lib/Components/Consignments/Screens/FinalSubmissionQuestions.tsx]: I bet the `Boolean` one might work, the later didnt'

File: src/lib/Components/Consignments/Screens/FinalSubmissionQuestions.tsx (hunk 2 above)
Comment (orta) [inline src/lib/Components/Consignments/Screens/FinalSubmissionQuestions.tsx]: The `phone-pad` only allows numbers to be inputted., but it looks like the edition number will need to be a string - so I'll change that 👍

**Comment by orta** [comment]
Alright 👍 feedback addressed, thanks!

**Comment by orta** [comment]
Ah yeah, by the time Peril ran the branch had been deleted, need to take that into account

**Pull request merged**